File: mod1/money.c
#include <stdio.h>
#include <math.h>

int main() {
    double cents;
    int nickel = 5;
    int dime = 10;
    int penny = 1;
    int quarter = 25;
    int count = 0;
    int number = 0;

    printf("Insert the cents: ");
    scanf("%lf", &cents);

    number = round(cents * 100);

    while (number > 0) {
        if (number >= quarter) {
            number -= quarter;
            count++;
        } else if (number >= dime) {
            number -= dime;
            count++;
        }else if (number >= nickel) {
            number -= nickel;
            count++;
        } else if (number >= penny) {
            number -= penny;
            count++;
        }
    }

    printf("Number of coins needed: %d\n", count);

    return 0;
}

File: mod6/money.py
cents = input("Enter your cents ")
count = 0
quarter = 25
dem = 10
nickel = 5
penny = 1
coin = round(float(cents) * 100)

while coin > 0:
 if coin >= quarter:
    coin -= quarter
    count = count + 1
 elif coin >= dem:
    coin -= dem
    count = count + 1
 elif coin >= nickel:
    coin -= nickel
    count = count + 1
 elif coin >= penny:
    coin -= penny
    count = count + 1

print("The number of coins is ", count)